.gitignore

```diff
@@ -35,3 +35,4 @@ coverage
 
 # Env
 .env
+package-lock.json
```

Comment:
I don't think that is a good idea to have two different lockfiles.

Reply:
I understood this ignore as a way to disregard the npm lockfile as the lockfile should always be stored in git as we do for `yarn.lock`

Reply:
You are right, removed the line from .gitignore.